lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1829,6 +1862,8 @@ SPIRVValue *LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II,
     return MemoryAccess;
   };
 
+  // Before adding intrinsic here, consider registering it at
```

Comment:
I think with this patch we have to "register" known intrinsics at the two places at the same time.

Reply:
Yes. I did not find a satisfactory solution without this.